cmd/cosign/cli/policy_init.go

```diff
@@ -114,6 +115,10 @@ func initPolicy() *cobra.Command {
 			root.Roles["root"] = role
 			root.Namespace = o.ImageRef
 
+			if o.Expires > 0 {
+				root.Expires = time.Now().Add(time.Duration(o.Expires) * 24 * time.Hour).UTC().Round(time.Second)
```

Comment:
You can use `AddDate` which takes a time parameter instead of multiplying no `24 * time.Hour`. 

`time.Now().AddDate(0, 0, 21).UTC()`